jpsonic-main/src/main/java/com/tesshu/jpsonic/service/PlayerService.java

```diff
@@ -96,9 +93,6 @@ public void init() {
         writeLock(playerLock);
         try {
             playerDao.deleteOldPlayers(60);
-            if (!settingsService.isUseExternalPlayer()) {
-                resetExternalPlayer();
```

Comment:
ExternalPlayer is already "OldPlayer" so it will need to be deleted regardless of its expiration date.

Reply:
Done.